OpenRA.Mods.Common/Traits/BotModules/BuildingRepairBotModule.cs

```diff
@@ -0,0 +1,42 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Mods.Common.AI;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Manages AI repairing base buildings.")]
+	public class BuildingRepairBotModuleInfo : ConditionalTraitInfo
```

Comment:
It would be cool to expand this in the future with options for starting/stopping repairs based on current funds, and repairing ally structures.

Reply:
And customizable DamageState thresholds, maybe even per BuildingCommonName (indirectly giving priority to some building types over others).